Gruntfile.js

```diff
@@ -2,17 +2,25 @@
   grunt.loadNpmTasks('grunt-contrib-watch');
   grunt.loadNpmTasks('grunt-contrib-sass');
   grunt.loadNpmTasks('grunt-contrib-copy');
+  grunt.loadNpmTasks('grunt-aws-s3');
+  grunt.loadNpmTasks('grunt-prompt');
 
+  var aws = grunt.file.readJSON('aws-keys.json');
```

Comment:
I guess this keys file is needed because of `grunt-aws-s3`, hmph. Ideally the plugin should be able to use the keys that are held in the user's home folder (in `~/.aws`), like the rest of the standard aws tools. Maybe there's another s3 module that could do this?